lib/ruby_lsp/requests/support/rubocop_diagnostic.rb

```diff
@@ -61,11 +61,17 @@ def to_lsp_code_action
 
         sig { returns(LanguageServer::Protocol::Interface::Diagnostic) }
         def to_lsp_diagnostic
+          severity, message = if @offense.correctable?
+            [RUBOCOP_TO_LSP_SEVERITY[@offense.severity.name], @offense.message]
+          else
+            [LanguageServer::Protocol::Constant::DiagnosticSeverity::ERROR,
```

Comment:
Since this is just a linting violation that can't be auto-corrected, let's use warning instead of error. Errors are typically for things that will really break your program, like a syntax error.

```suggestion
            [LanguageServer::Protocol::Constant::DiagnosticSeverity::WARNING,
```